lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -373,7 +383,7 @@ defmodule ExUnit.DocTestTest do
   test "doctest failures" do
     # When adding or removing lines above this line, the tests below will
     # fail because we are explicitly asserting some doctest lines from
-    # ActuallyCompiled in the format of test/ex_unit/doc_test_test.exs:<LINE>.
+    # ActuallyCompiled in the format of: test/ex_unit/doc_test_test.exs:<LINE>
```

Comment:
we can do that, I just thought is less noise when you use colons at the end of the line